### PR TITLE
Update/nw18

### PR DESCRIPTION
**Pull request by vzamanillo**
Changes proposed in this pull request:
- Use the new nw object instead the nw.gui wrapper library due to future deprecation.
- Fixed node-webkit-fdialogs opening error due to new NW.js separated context.

I've read the [Contributor License Agreement](/CONTRIBUTING.md#contributor-license-agreement)

### CODE REVIEW

File: src/app/lib/streamer.js [hunk 1]
```diff
@@ -48,7 +48,7 @@
             // and when the selected lang or default lang is set
             // subtitleDownloading is needed cos this is called every 300ms
 
-            if (stateModel.get('streamInfo').get('torrent').defaultSubtitle && stateModel.get('streamInfo').get('torrent').defaultSubtitle !== 'none' && hasSubtitles && subtitles != null && engine.files[0] && !downloadedSubtitles && !subtitleDownloading) {
+            if (stateModel.get('streamInfo').get('torrent').defaultSubtitle && stateModel.get('streamInfo').get('torrent').defaultSubtitle !== 'none' && hasSubtitles && subtitles !== null && engine.files[0] && !downloadedSubtitles && !subtitleDownloading) {
```
Comment (xaiki) [inline src/app/lib/streamer.js]: please split into another PR

File: src/app/lib/streamer.js [hunk 2]
```diff
@@ -124,7 +124,7 @@
                 streamInfo.set(torrent);
 
                 // we need subtitle in the player
-                streamInfo.set('subtitle', subtitles != null ? subtitles : torrent.subtitle);
+                streamInfo.set('subtitle', subtitles !== null ? subtitles : torrent.subtitle);
```
Comment (xaiki) [inline src/app/lib/streamer.js]: please split into another PR

File: src/app/lib/views/movie_detail.js [hunk 3]
```diff
@@ -183,15 +183,9 @@
...
-                $('.star-container').addClass('hidden');
-                AdvSettings.set('ratingStars', false);
-            } else {
-                $('.number-container').addClass('hidden');
-                $('.star-container').removeClass('hidden');
-                AdvSettings.set('ratingStars', true);
-            }
+            $('.number-container').toggleClass('hidden');
+            $('.star-container').toggleClass('hidden');
+            AdvSettings.set('ratingStars', $('.number-container').hasClass('hidden'));
```
Comment (xaiki) [inline src/app/lib/views/movie_detail.js]: great stuff, please split into another PR

File: src/app/lib/views/show_detail.js [hunk 4]
```diff
@@ -286,30 +286,24 @@
...
-                $('.star-container-tv').addClass('hidden');
-                AdvSettings.set('ratingStars', false);
-            } else {
-                $('.number-container-tv').addClass('hidden');
-                $('.star-container-tv').removeClass('hidden');
-                AdvSettings.set('ratingStars', true);
-            }
+            $('.number-container-tv').toggleClass('hidden');
+            $('.star-container-tv').toggleClass('hidden');
+            AdvSettings.set('ratingStars', $('.number-container-tv').hasClass('hidden'));
```
Comment (xaiki) [inline src/app/lib/views/show_detail.js]: great stuff, please split

**Comment by vzamanillo** [comment]
Thank you xaiki, splitted PR into this one, #454 and #455

**Comment by xaiki** [comment]
haha. i didn't realize you were PRing to the `update/nw18` branch sorry...

**Comment by vzamanillo** [comment]
Don't worry!
